core/server/src/main/java/alluxio/master/journal/JournalWriter.java

```diff
@@ -50,8 +52,14 @@
   private final String mJournalDirectory;
   /** Absolute path to the directory storing all completed logs. */
   private final String mCompletedDirectory;
+  /** Absolute path to the checkpoint file. */
+  private final String mCheckpointPath;
   /** Absolute path to the temporary checkpoint file. */
   private final String mTempCheckpointPath;
+  /** Absolute path to the backup checkpoint file. */
+  private final String mBackupCheckpointPath;
+  /** Absolute path to the temporary backup checkpoint file. */
+  private final String mTempBackupCheckpointPath;
```

Comment:
hrmmm, maybe there should be a javadoc somewhere that describes this process. There is a temp, backup, and tempbackup file, which can be confusing.